fastexcel-writer/src/test/java/org/dhatim/fastexcel/Correctness.java

```diff
@@ -519,4 +523,69 @@ public void canHideColumns() throws Exception {
         assertFalse("Column 4 should be visible", xws.isColumnHidden(4));
     }
 
+    @Test
+    public void shouldSetAllDefaultSheetProtectionOptions() throws IOException {
+
+        byte[] data = writeWorkbook(wb -> {
+            Worksheet ws = wb.newWorksheet("Worksheet 1");
+            ws.protect("HorriblePassword");
```

Comment:
It would be nice to [validate the password with Apache POI](https://poi.apache.org/apidocs/dev/org/apache/poi/xssf/usermodel/XSSFSheet.html#validateSheetPassword-java.lang.String-).

Reply:
Great! I will update the PR.